start_esp/start_esp.py

```diff
@@ -44,6 +44,7 @@
 # Location of NGINX template
 NGINX_CONF_TEMPLATE = "/etc/nginx/nginx-auto.conf.template"
 SERVER_CONF_TEMPLATE = "/etc/nginx/server-auto.conf.template"
+SERVER_CONF = "/etc/nginx/server_config.pb.txt"
```

Comment:
Please add comment
// Custom nginx config used by customers are hardcoded to this path